src/Json/JsonParser.php

```diff
@@ -183,6 +186,11 @@ public function parseStatusResult($jsonString) {
             $result->setTransactionSubType($json['transactionSubType']);
         }
 
+        if (isset($json['tracingData'])) {
+            $tracingData = $this->parseTracingData($json['tracingData']);
+            $result->setTracingData($tracingData);
+        }
```

Comment:
Maybe you could add another method which handles all this, since it was done twice and if something changes in that logic, you only have to apply the changes once.